src/components/ParticipantList.vue

```diff
@@ -251,32 +250,21 @@ Licensed under the Elastic License 2.0. */
   }
 
   const createParticipant = (amount: number): void => {
-    const i = amount || 1;
     const newParticipants: Participant[] = [];
-    // starWarsNames.all.length = 93
-    const openNames: string[] = starWarsNames.all.filter(
-      (n) => !participantsList.value.find((p) => p.alias === n),
-    );
 
-    const addNew = (alias: string): number =>
+    const maxId = participantsList.value.reduce(
+      (max: number, current: Participant | undefined): number => {
+        const pId = current?.participantId || 0;
+        return pId > max ? pId : max;
+      },
+      participantsList.value?.length || 0,
+    );
+    Array.from({ length: amount || 1 }).forEach((_, i) => {
```

Comment:
das `amount || 1` bring einfach auch clutter und macht den code unreadable. Amount ist ja kein optional param, der sollte doch immer gesetzt sein.

Reply:
Hmm, auch das du ein Array erzeugst damit du durch loopen kannst mit einer anzahl ist etwas unnötig. Warum nicht einfach eine for loop? Somit vermeidest auch die +1 Konstante weil es bei index 0 startet. 

```
for (let i = 1; i <= amount; i++) {
  newParticipants.push({
    alias: `P ${maxId + i}`,
    studyId: props.studyId,
  });
}
```